reactioncommerce/reaction-next-starterkit
# feat: Add variant list components

Add variant list components

### Testing
1. Add multiple variants to a product
2. Verify all variants are rendered
3. Click on a variant and verify URL updates accordingly.
